packages/rancher-istio-1.8/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: local
 packageVersion: 00
-releaseCandidateVersion: 01
+releaseCandidateVersion: 03
```

Comment:
Why not rc 02 on both of these ?

Reply:
I was being optimistic that https://github.com/rancher/charts/pull/1132 would get merged first